package.json

```diff
@@ -24,7 +24,7 @@
   },
   "lint-staged": {
     "*.{js,ts,jsx,tsx}": [
-      "eslint --fix --ignore-path .gitignore"
+      "eslint --fix --ignore-path .gitignore --max-warnings=0"
```

Comment:
Although, I'm not sure if it makes sense for these to fail on staged but not on `yarn lint` (eg not fail on CI). I feel like those should match. This will have no change in terms of enforcing no warnings in the codebase, since precommit hooks are an optional part of the dev setup. What do you think?

If there are specific warnings that we want to be errors (eg missing hook dependencies) then we can mark them as errors in eslintrc.

Reply:
Ok yes, that makes sense to me. Warning should stay warnings, but anything that is always going to be a blocking comment on a PR (unless it has been explicitly ignored for a specific reason) should be elevated to an error.